.scrutinizer.yml

```diff
@@ -9,27 +9,117 @@ before_commands:
 tools:
     external_code_coverage:
         timeout: 3600
-    php_code_coverage:
-        enabled: true
-    php_code_sniffer:
-        enabled: false
-    php_cpd:
-        enabled: true
-        excluded_dirs: ["bin", "docs", "tests", "tools", "vendor"]
-    php_cs_fixer:
-        enabled: false
-    php_loc:
-        enabled: true
-        excluded_dirs: ["bin", "docs", "tests", "tools", "vendor"]
-    php_mess_detector:
-        enabled: true
-        filter:
-            paths: ["lib/*"]
-    php_pdepend:
-        enabled: true
-        excluded_dirs: ["docs", "examples", "tests", "vendor"]
-    php_analyzer:
-        enabled: true
-        filter:
-            paths: ["lib/*", "tests/*"]
-    sensiolabs_security_checker: true
+
+filter:
+    excluded_paths:
+        - docs
+        - tools
+
+checks:
+    php:
+        code_rating: true
```

Comment:
The point is having *LESS* flags to maintain. Let's keep all the defaults we can keep, and only override the ones we need to override.

This project will never reach "object calisthenics" quality within our lifetime, so it's useless to use a code quality analysis tool to drive improvements: we are good at reviewing stuff.